tidb-monitoring-api.md

```diff
@@ -6,15 +6,15 @@ aliases: ['/docs/dev/tidb-monitoring-api/']
 
 # TiDB Monitoring API
 
-You can use the following two types of interfaces to monitor the TiDB cluster state:
+You can use the following types of interfaces to monitor the TiDB cluster status:
 
-- [The state interface](#running-status): this interface uses the HTTP interface to get the component information.
+- [The status interface](#running-status): this interface uses the HTTP interface to get the component information.
 - [Storage information](#storage-information): this interface uses the HTTP interface to get the storage information of data tables.
```

Comment:
Is this one parallel with the other two? If yes, it would be better to use similar words such as "The xxx interface", and adjust the body heading level. Please help confirm. Thanks!

Reply:
As confirmed with @xhebox, the storage information is subordinate to the status interface, together with the running status. Therefore, the storage information should not be in parallel with the status interface. PTAL. Thanks!